src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -152,6 +146,13 @@
       <_sourcePath Include="@(_sourcePathCandidate)" Condition="Exists('%(Identity)')" />
     </ItemGroup>
 
+    <!-- Add versions file with the hashes of the repos we consume -->
+    <ItemGroup>
+      <FilesToPackage Include="$(IntermediateOutputPath)\Microsoft.NETCore.App.versions.txt">
+        <TargetPath></TargetPath>
```

Comment:
It didn't happen before because @ericstj used to have a temp item for these files where TargetPath was set `_FilesToPackage`, but in a recent PR he moved to use the one without the underscore which made it such that my version file was impacted as well.